### DISCUSSION TITLE
Opt-in to Hacktoberfest

### DISCUSSION THREAD

**Discussion by equal-l2** [Homebrew/discussions#26] [Everyday usage] [answered]
I saw [Hacktoberfest is now opt-in only](https://hacktoberfest.digitalocean.com/hacktoberfest-update) for repos which have "hacktoberfest" topic.
Will Homebrew participate in Hacktoberfest, by adding "hacktoberfest" topic?

**Comment by jonchang** [comment, marked as answer]
I've added the `hacktoberfest` topic to a few of the more active Homebrew repositories like homebrew-core and homebrew-cask. We haven't seen too much spam yet this year so hopefully this month goes by without too much of a problem.

**Comment by equal-l2** [comment]
Thank you, @jonchang!
I hope everyone earns their swags through productive contributions to Homebrew.
